tutor/src/components/navbar/user-actions-menu.jsx

```diff
@@ -100,6 +100,8 @@ export default class UserActionsMenu extends React.PureComponent {
 
     const course = Courses.get(this.props.courseId);
     const isConceptCoach = get(course, 'is_concept_coach', false);
+    const firstName = User.name ? User.name.replace(/ .*/, '') : '';
+
```

Comment:
How about we add a method to User for getting the first name.  Probably something like:

```js
@computed get firstName() {
  return this.first_name || User.name.replace(/ .*/, '') || '';
}
```

Then we can just use `User.firstName` from the jsx below.